drivers/sensor/lis2dw12/Kconfig

```diff
@@ -68,6 +68,94 @@ config LIS2DW12_INT_PIN_2
 	bool "int2"
 endchoice
 
+menuconfig LIS2DW12_PULSE
+	bool "Pulse detection"
+	help
+	  Enable pulse (single/double tap) detection
+
+if LIS2DW12_PULSE
+
+choice
+	prompt "Pulse interrup source"
+	default LIS2DW12_ONLY_SINGLE
+
+config LIS2DW12_ONLY_SINGLE
+	bool "single"
+
+config LIS2DW12_SINGLE_DOUBLE
+	bool "single/double"
+
+endchoice
+
+config LIS2DW12_PULSE_THSX
+	hex "Pulse X-axis threshold"
+	range 0 0x1F
+	default 0x0E
```

Comment:
just a question. Did you select all the default values based on some sort of calculation?

Reply:
Unfortunately these values were obtained experimentally during work under my specific application.
All these values have to be adjust for any individual case based on many dependencies (e.g.: interrupt sensitivity user wants to achieve or chosen ODR value).

On the other hand values proposed as defaults should be a good start for further investigation.